scripts/pr_acctest_basic.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+all_files=$(git diff $1 $2 --name-only huaweicloud | grep -v "_test.go")
```

Comment:
the diff result will be very huge if the branch based on an outdated upstream

Reply:
done